packages/espressocash_app/pubspec.yaml

```diff
@@ -84,7 +85,7 @@ dev_dependencies:
   flutter_driver:
     sdk: flutter
   flutter_gen_runner: ^5.4.0
-  flutter_native_splash: ^2.3.8
+  flutter_native_splash: ^2.3.7
```

Comment:
Why does it need to be downgraded?

Reply:
it has dependency conflicts when adding `stellar_flutter_sdk`. the cli recommends downgrading to that version to resolve